src/citrine/informatics/workflows/design_workflow.py

```diff
@@ -108,6 +109,8 @@ def _post_dump(self, data: dict) -> dict:
     @property
     def executions(self) -> WorkflowExecutionCollection:
```

Comment:
We can use the decorator version here, right?  That will make it show up in some IDEs, I think